coinmarketcap-client/src/lib.rs

```diff
@@ -0,0 +1,62 @@
+use anyhow::Result;
```

Comment:
I took this package from [here](https://github.com/p2p-org/token-service/blob/main/price-service/src/sources/coinmarketcap/client.rs)